src/main/java/com/start/learn/practicewithstrings/PracticeWithStrings.java

```diff
@@ -5,82 +5,40 @@
 
 public class PracticeWithStrings {
 
-    public static void lettersAndSymbolsRemover(String str) {
-        StringBuilder symbolsRemover = new StringBuilder(str);
-        symbolsRemover.deleteCharAt(str.length() - 1);
-        symbolsRemover.deleteCharAt(0);
-        symbolsRemover.toString();
-        System.out.println("Phrase where removed first and last letters : ");
-        System.out.println(symbolsRemover);
-    }
-
-    public static void replacingEnteredPhraseWithSymbols(String inputPhrase) {
-        StringBuilder replacer = new StringBuilder(inputPhrase.length());
-        for (int i = 0; i < inputPhrase.length(); i++) {
-            replacer.append('\"');
+    public static int getLettersAmount(String originalPhrase, char letterOrSymbolInput) {
+        if (Character.isLetter(letterOrSymbolInput)) {
+            originalPhrase = originalPhrase.toLowerCase(Locale.ROOT);
```

Comment:
remove Locale.ROOT-> it makes no sence here

Reply:
done